app/actions/navigation.js

```diff
@@ -14,6 +14,16 @@ export const goToContactCreate = () => {
   }
 }
 
+export const goToContactEdit = (contactSid, goBack = false) => {
+  return {
+    payload: {
+      contact_sid: contactSid,
+      go_back: goBack
```

Comment:
`shouldGoBack` or `willGoBack` instead since it's boolean?

Reply:
Yeah, it should be `shouldGoBack`. Will include this in #102